MonoGame.Framework.Content.Pipeline/Serialization/Compiler/TextureCubeWriter.cs

```diff
@@ -20,6 +20,8 @@ protected internal override void Write(ContentWriter output, TextureCubeContent
             if (!level0.TryGetFormat(out format))
                 throw new Exception("Couldn't get format for TextureCubeContent.");
 
+            format = value.GetActualFormat(format);
```

Comment:
Good point - if we know for sure that sRGB isn't supported by the target platform, we should fail at content build time.

But we'll still have to support the case where textures are built with sRGB=true, but it isn't supported at runtime (i.e. once we implement it for OpenGL, not all graphics cards will necessarily support the required extensions). In that case, we'll have to fallback to using a non-sRGB texture format.